### ISSUE TITLE
Update Second player screen when First player moves 

### ISSUE THREAD

**Issue by amarkpark** [FallGambit/fall_gambit#61]
I am currently testing the jg/turn_logic feature branch (this is NOT a bug in the branch, this is just by way of letting anyone reading this know what version of code I am running for the purposes of tracking/reproduction.) When I move as one player in one browser that browser updates with the state of the current turn, but the second browser with the second player does NOT update to show the state of the current turn until that player refreshes their screen. We need to force the views to update for BOTH players whenever the user_turn field is updated at the completion of a turn, or else start a listener that will run on every machine that accesses an active game and queries the user_turn field for changes and then triggers a reload. Or potentially just have a timer trigger a reload evert so often.  Obviously some of these options are more or less efficient than others but we have to find a way to alert the inactive player when the active player completes a move.

**Comment by jeffgerlach** [comment]
Agreed. I propose we just poll the server and refresh the game page (maybe every 5-10 seconds?) for a quick fix, while @nicolaguidi has taken the task to investigate using websockets to update the game when a move is actually made.
